From ralf@linux-mips.org Thu Aug 23 11:45:52 2007
From: Ralf Baechle <ralf@linux-mips.org>
Date: Thu, 23 Aug 2007 19:45:49 +0100
Subject: PCI: Remove __devinit from pcibios_get_irq_routing_table
To: linux-kernel@vger.kernel.org, Andrew Morton <akpm@linux-foundation.org>, gregkh@suse.de, linux-pci@atrey.karlin.mff.cuni.cz
Message-ID: <20070823184549.GA21650@linux-mips.org>
Content-Disposition: inline


pcibios_get_irq_routing_table is an exported symbol.  This results in a
modpost warning:

WARNING: vmlinux.o(.text+0xdca51): Section mismatch: reference to .init.text:pcibios_fixup_bus (between 'pci_scan_child_bus' and 'pci_scan_bus_parented')

Signed-off-by: Ralf Baechle <ralf@linux-mips.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 arch/i386/pci/pcbios.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/arch/i386/pci/pcbios.c
+++ b/arch/i386/pci/pcbios.c
@@ -412,7 +412,7 @@ struct irq_routing_options {
 	u16 segment;
 } __attribute__((packed));
 
-struct irq_routing_table * __devinit pcibios_get_irq_routing_table(void)
+struct irq_routing_table * pcibios_get_irq_routing_table(void)
 {
 	struct irq_routing_options opt;
 	struct irq_routing_table *rt = NULL;
